Integrations/MicrosoftTeams/MicrosoftTeams.yml

```
@@ -54,7 +54,7 @@ configuration:
   name: longRunning
   required: false
   type: 8
-- display: Port mapping, e.g. <host port>:<docker port> (Required for investigation
+- display: Port mapping, e.g. <port> or <host port>:<docker port> (Required for investigation
```

Let's remove the host:docker port option for this description. We can have this in the more detailed description. I think it will save confusions.
```suggestion
- display: Listen port, e.g. 7000 (Required for investigation
```